services/app-api/src/postgres/postgresStore.ts

```diff
@@ -67,6 +69,16 @@ type Store = {
     findUser: (id: string) => Promise<User | StoreError>
 
     insertUser: (user: InsertUserArgsType) => Promise<User | StoreError>
+
+    updateUserAssignedState: (
+        userID: string,
+        state: State
```

Comment:
I’m hesitant for us to have the Prisma types in the type signatures here. The idea behind our doman models is that they are the types used for calls between modules, which lets us separate our implementation at the prisma layer from everywhere else we use them. When you pick this up can we pair a little and work out how we want to handle this? I appreciate that ts is duck typed so for the easy case where the types line up we don’t have to manually convert even if we do have to duplicate the types.